scripts/lib/CIME/SystemTests/pre.py

```diff
@@ -31,26 +33,36 @@ def __init__(self, case):
                                        run_two_suffix='pr',
                                        run_one_description='no pause/resume',
                                        run_two_description='pause/resume')
+        self._stopopt  = 'ndays'
```

Comment:
They were really placeholders. I modified the values to just initialize the variables in `__init__`, the working values do come from config_tests.xml.